src/i965_encoder.c

```diff
@@ -1453,6 +1468,16 @@ intel_enc_hw_context_init(VADriverContextP ctx,
             encoder_context->quality_range = ENCODER_QUALITY_RANGE_AVC;
         } else
             encoder_context->quality_range = ENCODER_QUALITY_RANGE;
+
+        if (obj_config->entrypoint == VAEntrypointFEI) {
+            encoder_context->fei_enabled = 1;
```

Comment:
Can disable fei_enabled when HAS_FEI_H264_ENCODING(i965) is not true, then not need to check Is_SKL() in gen9_avc_encode.c for FEI case.